### PR TITLE
add blood type select

### LINKED ISSUE #75: Form element for blood types
We need a dropdown selection for the blood types below. This input should also be integrated into our patient data form as an optional field.

The rhesus factor is the + or - at the end of the blood group. The last four options are only blood groups without the rhesus factor, in case this information is not available. In the background, the select should output a structure where the blood group and rhesus factor is seperate.

- Blood types
  - A+
  - A-
  - B+
  - B-
  - AB+
  - AB-
  - O+
  - O-
  - A
  - B
  - AB
  - O

### PR DESCRIPTION
**Pull request by DasProffi**
closes #75 

dropdown
Blood Type Select

### CODE REVIEW

File: lib/l10n/app_de.arb [hunk 1]
```diff
@@ -30,5 +30,15 @@
   "notify": "Benachrichtigen",
   "notifyCancel": "Benachrichtigung zurücknehmen",
   "startNavigation": "Navigation starten",
-  "message": "Nachricht"
-}
\ No newline at end of file
+  "message": "Nachricht",
+  "bloodType": "Blut-Typ",
```
Comment (Florian-Sabonchi) [inline lib/l10n/app_de.arb:34]: I would suggest as translation "Blutgruppe"

**Review by Florian-Sabonchi — COMMENTED** [review]
In my opinion, it would be useful to include the blood group in the patient data. What do you think  @MaxSchaefer

**Comment by MaxSchaefer** [comment]
> In my opinion, it would be useful to include the blood group in the patient data. What do you think @MaxSchaefer

Yes, of course. You (@Florian-Sabonchi) can merge #81 into main. @DasProffi should then be able to merge main into this branch.

**Comment by MaxSchaefer** [comment]
I don't like the two-selector approach.

> ... Let's have one select with the upper options and handle the blood group and rhesus factor in the background. ...

https://github.com/helpwave/mobile-app/issues/75#issuecomment-1361764097

This means, one select with options like A+, A-, B+ ... . In the code, this select should output a structure with the blood type (A, B ...) and rhesus factor (+/-) separated.

What do you think about this?

**Comment by DasProffi** [comment]
I agree that the two selector approach isn't perfect, but a select with 8 or 12 twelve options seemed even more confusing for a user. 

Maybe a double select window would be the best way, but it would require probably some more complex custom widget
doubleSelect

**Comment by use-to** [comment]
Could you provide updated screenshots, please?

**Comment by florian-sabonchi** [comment]
> Could you provide updated screenshots, please?

image

**Review by use-to — APPROVED** [review]

**Pull request merged**